src/init.cpp

```diff
@@ -289,6 +293,10 @@ void Shutdown()
     GetMainSignals().UnregisterWithMempoolSignals(mempool);
     g_wallet_init_interface.Close();
     globalVerifyHandle.reset();
+
+    delete pzcashParams;
```

Comment:
if you declare as a `std::unique_ptr` above you don't have to manually invoke the destructor here